### PR TITLE
Assignment3

### PR DESCRIPTION
**Pull request by rutaihwa**
#79  Testing the app, probably tests should be more robust. Not sure about the Makefile, i just got it ast it is from hapi package.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -4,25 +4,23 @@ var Hapi = require('hapi');
...
+    }
 
     var server = new Hapi.Server();
-    server.connection({ port: 8000 });
+    server.connection({ port: port });
     server.register(Version, function (err) {
 
-        Hoek.assert(!err, err);
-        server.start(function (err) {
+	Hoek.assert(!err, err);
```
Comment (chyld) [inline lib/index.js]: don't assert, just pass err back to caller

File: start.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Load modules
+
+var Hoek = require('hoek');
+var lib = require('./lib');
```
Comment (chyld) [inline start.js]: call it Server instead of lib ... it's more descriptive. Lib means nothing.

Comment (rutaihwa) [inline start.js]: @chyld, that makes sense..

File: lib/index.js (hunk 1 above)
Comment (rutaihwa) [inline lib/index.js]: sweet, let the caller handle errors.

**Pull request closed without merging**